RWKV-v4neo/src/trainer.py

```diff
@@ -18,6 +18,9 @@ class train_callback(pl.Callback):
     def __init__(self, args):
         super().__init__()
         self.args = args
+    
+    def on_train_start(self, trainer, pl_module):
```

Comment:
These things remained unchanged compared to BlinkDL's original implementation. I don't expect to change those in this fork; you might want to submit another PR to the upstream repository.